server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -97,7 +97,24 @@ export function create(
                 crypto.randomBytes(4).toString("hex"),
                 values);
 
-            handleResponse(createP.then(() => id), response, undefined, 201);
+            // Generate creation token given a jwt from header
+            const authorizationHeader = request.header("Authorization");
+            const tokenRegex = /Basic (.+)/;
+            const tokenMatch = tokenRegex.exec(authorizationHeader);
+            const token = tokenMatch[1];
+
+            const tenantKeyP = tenantManager.getKey(tenantId);
+
+            handleResponse(Promise.all([createP, tenantKeyP]).then(([_, key]) => {
+                return generateToken
+                    ? {
+                        id,
+                        token: getCreationToken(token, key, id),
+                    }
+                    : {
+                        id,
```

Comment:
My apologies, I asked you to convert this to an object. Would you mind switching it back to just the ID string? It slipped my mind that we need to remain backward compatible to old driver versions that can't handle an object response here 🤦‍♂️